src/client/components/VesselDetails.js

```diff
@@ -42,4 +43,16 @@ function VesselDetails({
   );
 }
 
+VesselDetails.propTypes = {
+  vesselName: PropTypes.string,
+  departsFrom: PropTypes.string,
+  arrivesAt: PropTypes.string,
+  etd: PropTypes.string,
```

Comment:
Soon to be changed to `departure_time`. Keep a look out if this ends up being merged before or after Nayanas database refactor.

Reply:
Ok